content/registry/packages/aws-apigateway/how-to-guides/aws-apigateway-go-routes.md

```diff
@@ -2,6 +2,7 @@
 title: "Routes in API Gateway | Go"
```

Comment:
thoughts on having the title and meta description the same? instead of the or bar?

Reply:
ah no, cause those need to be 60 characters are less, will leave as is